v3/integrations/nrlambda/handler.go

```diff
@@ -67,14 +67,36 @@ func responseEvent(ctx context.Context, event interface{}) {
 	}
 }
 
+type writerProvider interface {
+	borrowWriter(needsWriter func(writer io.Writer))
+}
+
+type defaultWriterProvider struct {
+}
+
+func (wp *defaultWriterProvider) borrowWriter(needsWriter func(io.Writer)) {
+	// If the telemetry named pipe exists and is writable, use it instead of stdout
+	const telemetryNamedPipe = "/tmp/newrelic-telemetry"
```

Comment:
Should we make this a global const so it doesn't need to be redefined each time?

Reply:
Doesn't the compiler do that for consts?

Reply:
Oh, that I'm not sure about! Good question, I'll do some research.

Reply:
Ah ha! You are correct! I learned something today! https://qvault.io/2019/10/14/constants-in-go-vs-javascript-and-when-to-use-them/